index.html

```diff
@@ -1,26 +1,100 @@
-
+<!-- 
 Course Resources Useful tools
 
-<!-- First link -->
+ First link 
 https://loremipsumgenerator.com/generator/?n=1&t=p Lorem ipsum generator
 
-<!-- Second link  -->
+ Second link  
 https://micro-editor.github.io/ Micro: Terminal Text Editor HTML
 
-<!-- First Link -->
+ First Link 
 http://www.google.com/search?q=html%20is%20awesome Look for HTML
 
-<!-- Second link -->
+ Second link 
 https://developer.mozilla.org/en-US/docs/Web/HTML MDN HTML portal
 
-<!-- Third Link -->
+ Third Link 
 https://developer.mozilla.org/en-US/docs/Web/HTML/Element MDN HTML element reference
 
-<!-- Fourth Link -->
+ Fourth Link 
 https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references HTML Entity reference table
 
-<!-- Fifth Link -->
+Fifth Link 
 https://validator.w3.org/#validate_by_upload+with_options HTML validator
 
-<!-- Sixth Link -->
-https://developer.mozilla.org/en-US/docs/Learn MDN HTML Tutorial
\ No newline at end of file
+ Sixth Link 
+https://developer.mozilla.org/en-US/docs/Learn MDN HTML Tutorial -->
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Basic HTML Exercises</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="post">
```

Comment:
instead of using to divs you could also use one div with two classes e.g <div class="container post"></div>